modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: Procedure
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
+
+The link:https://github.com/openshift/console-plugin-template[plugin template] demonstrates how you can translate messages with link:https://www.i18next.com/[react-i18next].
+
+
+.Prerequisites
+* You must have the plugin template cloned locally.
+* You must be running the {product-title} web console in a container. You can use either Docker or Podman 3.2.0+.
+* You must have `oc` installed.
+
+.Procedure
+
+. Prefix the name with `plugin__` to avoid any naming conflicts. For example, the plugin template uses the `plugin__console-plugin-template` namespace. You can use the `useTranslation` hook, for example:
```

Comment:
While it does avoid conflicts, we actually require the `plugin__` prefix. The messages won't load otherwise.

I'd try make it clear that the `plugin__console-plugin-template` is only the default and must be updated when you rename your plugin. (This is mentioned below, but still a little unclear in my opinion. It might be good to give an example like `plugin__my-plugin`.)